src/EventStore.Client.Streams/EventStoreClient.cs

```diff
@@ -45,9 +45,6 @@ public partial class EventStoreClient : EventStoreClientBase {
 							?.Value!, ex),
 			};
 
-		public EventStoreClient(IOptions<EventStoreClientSettings> options) : this(options.Value) {
```

Comment:
This would be a breaking change if anyone were using this?

Reply:
Yes and no... typically you don't call this constructor, it is done with DI magic.